client/marketing/overview/style.scss

```diff
@@ -2,6 +2,10 @@
 	max-width: 1032px;
 	margin: 0 auto;
 
+	.components-card {
+		margin-bottom: 24px;
+	}
+
 	.woocommerce-card__header {
 		padding: 16px 24px;
 		border-bottom: 1px solid $gray-100;
```

Comment:
Looks like these `.woocommerce-card__*` styles aren't being used any more and should be removed.